src/components/ProductListCard/ProductListCard.js

```diff
@@ -0,0 +1,152 @@
+import React, { useState, useEffect } from 'react';
+import './ProductListCard.scss';
+import { useNavigate, Link } from 'react-router-dom';
+
+const ProductListCard = props => {
+  const [change, setChange] = useState(true);
+
+  const navigate = useNavigate();
+  const { data } = props;
+  const goToDetail = id => {
+    navigate(`/productdetail/${id}`);
+  };
+
+  const clickLike = (id, isLike) => {
+    console.log(id);
+    console.log(isLike);
+    if (!localStorage.getItem('accessToken')) {
+      return alert('로그인이 필요합니다.');
+    }
+
+    if (props.isLike === false) {
+      fetch('http://10.58.52.96:8000/users/signup', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json;charset=utf-8',
+        },
+        body: JSON.stringify({
+          id: props.id,
+          accessToken: '123',
+        }),
+      }).then(() => {
+        setChange(!change);
+      });
+    } else {
+      fetch('http://10.58.52.96:8000/users/signup', {
+        method: 'DELETE',
+        headers: {
+          'Content-Type': 'application/json;charset=utf-8',
+        },
+        body: JSON.stringify({
+          id: props.id,
+          accessToken: '123',
+        }),
+      }).then(() => {
+        setChange(!change);
+      });
+    }
+  };
+
+  return (
+    <div className="productListCard">
+      <div
+        className="productListCardImgBox"
+        onClick={() => {
+          goToDetail(data.id);
+        }}
+      >
+        <img
+          className="picFirst"
+          src={data.productImg[1].url}
+          alt="cardImage"
+        />
+        <img
+          className="picSecond"
+          src={data.productImg[0].url}
+          alt="cardImage"
+        />
+        <div
+          className="picCartBox"
+          onClick={e => {
+            e.stopPropagation();
+            goToDetail('cart');
+          }}
+        >
+          <img
+            className="picCart"
+            src={process.env.PUBLIC_URL + '/images/shopping-cart.png'}
+            alt="cardImage"
+          />
+        </div>
+      </div>
+      <div className="titleBox">
+        <div className="labelBox">
+          {data.isNew ? <div className="newProduct">신제품</div> : null}
```

Comment:
isNew는 기본적으로 Boolean값을 가질 것으로 보입니다. true와 false를 비교할 때 false인 경우에 대해서 아무런 UI를 표기해주고 싶지 않을땐 &&연산자를 활용해서 아래와 같이 표기하는게 효율적입니다.
```suggestion
          {data.isNew && <div className="newProduct">신제품</div> }
```